### PR TITLE
Remove groupLabel parameter from popDebugGroup

### PR DESCRIPTION
**Pull request by bjjones**
Remove groupLabel parameter from `popDebugGroup`.

`popDebugGroup` was initially merged to the IDL in https://github.com/gpuweb/gpuweb/commit/f6b5b250101bf13ac3ed82810be1df247fc7a3f1

None of the backends allow you to pass the `groupLabel` as a parameter. All backends will match `popDebugGroup` equivalent with the last called `pushDebugGroup` equivalent.

**DX12 PIX**
`void PIXEndEvent(ID3D12GraphicsCommandList* commandList);` [[MSDN]](https://blogs.msdn.microsoft.com/pix/winpixeventruntime/)
    
**Vulkan**
`void vkCmdDebugMarkerEndEXT(VkCommandBuffer commandBuffer);` [[Khronos]](https://www.khronos.org/registry/vulkan/specs/1.0-extensions/html/vkspec.html#vkCmdDebugMarkerEndEXT)

**Metal**
`func popDebugGroup()` [[Apple]](https://developer.apple.com/documentation/metal/mtlcommandbuffer/2869549-popdebuggroup)

### CODE REVIEW

**Comment by kvark** [comment]
To be honest, when I first skimmed over #126 I thought that this is intentional: it's not infrequent for developers to mess up the push/pop without realizing it, and it's a good practice to comment on the pop() like:
> context.popDebugMarker(); //shadow maps

So making it a part of the API seems like it could be useful. HOWEVER. We don't need to be too creative here, and we are safe to leave those enhancements to whatever layers are atop of us :)

TL;DR: PR is fine

**Comment by magcius** [comment]
Yeah, this was my copy/paste typo in #121. Even my own polyfill of the API doesn't have that argument.
